<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserInfoWidget</class>
 <widget class="QWidget" name="UserInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>510</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>340</width>
    <height>510</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>340</width>
    <height>510</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>340</width>
    <height>510</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="professionLabel2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>340</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>计算机/互联网/通信</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="ownLabel1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>370</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(174, 174, 174);
font: 12px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>个人</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="signLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>281</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(174, 174, 174);
font: 14px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>给梦想一个成长的机会</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="nicknameLabel1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(174, 174, 174);
font: 12px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>昵称</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="emailLabel2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>310</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>ls56647426@foxmail.com</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="mobileLabel1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(174, 174, 174);
font: 12px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>手机</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="locationLabel1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>400</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(174, 174, 174);
font: 12px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>所在地</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="mobileLabel2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>13244434212</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="professionLabel1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>340</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(174, 174, 174);
font: 12px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>职业</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="nicknameLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>190</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>190</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>190</width>
     <height>50</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>190</width>
     <height>50</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 22px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>龙天行</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="usernameLabel1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(174, 174, 174);
font: 12px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>账号</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="emailLabel1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(174, 174, 174);
font: 12px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>邮箱</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ownLabel2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>370</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>21  男  双子座</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QWidget" name="infoTopWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>340</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#infoTopWidget{
	border-radius: 3px;
	border-image: url(:/Icon/topbg.png);
}</string>
   </property>
   <widget class="QPushButton" name="picBtn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/Icon/user.png);
background-color: rgb(218, 219, 221);
border-radius:40px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="changeHeadBtn">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>更换头像</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#changeHeadBtn{
	border-image: url(:/Icon/choose_pix1.png);
}
QPushButton#changeHeadBtn:hover{
	border-image: url(:/Icon/choose_pix2.png);
}

</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="nicknameLabel2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>250</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>龙天行</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QPushButton" name="editUserBtn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>95</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>修改资料</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#editUserBtn{
	border-image: url(:/Icon/edit.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="locationLabel2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>400</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>巴黎</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="usernameLabel2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>160</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12px &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>2356732870</string>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
